src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2140,6 +2140,7 @@ static void updateAttrInNotifyCer
           caP->previousValue = new ContextAttribute();
```

Comment:
Just above `if (targetAttr->valueType != ValueTypeNone)` include the following comment:

> /* If attribute has none value, then its value is not updated (neither previousValue) */

...although I'm not fully sure. What about?

```
PUT /v2/entities/E/attrs/A
{
   "type": "T",
   "value": null
}

It would work as expected?